bin/kafka-streams-application-reset.sh

```diff
@@ -18,4 +18,4 @@ if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
 
-exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.tools.StreamsResetter "$@"
+exec $(dirname $0)/kafka-run-class.sh kafka.tools.StreamsResetter "$@"
```

Comment:
Can we not keep the original package or is that not kosher in `core`? The fully qualified name of the tool is kind of public API, so if we want to move this back into `tools` eventually, ideally we could do so without having to change that name.